components/webflow/sources/new-deleted-collection-item/new-deleted-collection-item.mjs

```diff
@@ -0,0 +1,23 @@
+import common from "../common.mjs";
+
+export default {
+  type: "source",
+  key: "webflow-new-deleted-collection-item",
```

Comment:
What would be the reason behind replacing the name from `deleted-collection-item` to `new-deleted-collection-item`?

Reply:
Because source `Name` param should start with `New...`, so I changed the file names too.